compiler/src/dotty/tools/dotc/transform/InterceptedMethods.scala

```diff
@@ -44,11 +44,11 @@ class InterceptedMethods extends MiniPhase {
 
   override def phaseName: String = InterceptedMethods.name
 
-  private[this] var primitiveGetClassMethods: Set[Symbol] = _
+  private[this] var scalaValueClasses: scala.collection.Set[Symbol] = _
 
   /** perform context-dependant initialization */
   override def prepareForUnit(tree: Tree)(implicit ctx: Context) = {
-    primitiveGetClassMethods = Set[Symbol]() ++ defn.ScalaValueClasses().map(x => x.requiredMethod(nme.getClass_))
```

Comment:
So, the improvement is that before `prepareForUnit` returned all the symbols for `getClass` methods, by generating those stubsymbols that are needed, and now it doesn't?

Reply:
Yes. Mostly the cost was in creating the set.